src/snyk/common/views/issueTreeProvider.ts

```diff
@@ -109,6 +115,19 @@ export abstract class ProductIssueTreeProvider<T> extends AnalysisTreeNodeProvid
     return nodes;
   }
 
+  getBaseBranch(): TreeNode | null {
+    const deltaFindingsEnabled = this.configuration.getDeltaFindingsEnabled();
+
+    //TODO: get the actual base branch from Snyk Language Server
```

Comment:
This should just ask a config setting, what's the base branch for the folder. LS sends a folderConfig notification for all known workspace folders when they are registered at LS.